### ISSUE TITLE
Img tag missing alt text

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#708]
# a11y bug

Experienced in `/it/18app` page.
```
Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #projectLeadersIndicator > div > div > ul > li > a
   └── <a href="https://github.com/alranel" title="alranel" target="_blank"><img width="50" src="https://av...</a>
```
Please note: this is not a single page related bug but most probably the fix has to be provided in the layout that the page inherits.

**Comment by Roshan13046** [comment]
I would like to work on this issue.
Suggest some ways to get started.

**Comment by libremente** [comment]
Hi @Roshan13046 and thanks for your interest!
As you may see, that page https://github.com/italia/developers.italia.it/blob/master/_platforms/it/18app.md has "github_team" set so you should probably check into this file https://github.com/italia/developers.italia.it/blob/70152d5c1537d8a80d92d8bfd7da4f2ca44cf2a6/_includes/github_team_members.html and see if the problem lies there. I can see an img tag without the alt text so this should probably be fixed. Since we have "member.name" I'd put that as an alt text. WDYT? Were you able to setup the dockerized dev environment to test it locally? Thanks!

**Comment by Roshan13046** [comment]
I have started working on this issue.

**Comment by bfabio** [comment]
Hi @Roshan13046, are you still working on it?

**Comment by Roshan13046** [comment]
> Hi @Roshan13046, are you still working on it?

Yes, I am working on it.

**Comment by maurovitale** [comment]
Hi @libremente, if @Roshan13046  is not trying to fix this issue anymore, I checked the problem with devtools and I've the solution. The only problem is that the github_members file it's missing  from the project (it's part of .gitignore file), so I can't test the fix properly. Do you have any way to include it on the project ?

**Comment by Roshan13046** [comment]
> Hi @libremente, if @Roshan13046 is not trying to fix this issue anymore, I checked the problem with devtools and I've the solution. The only problem is that the github_members file it's missing from the project (it's part of .gitignore file), so I can't test the fix properly. Do you have any way to include it on the project ?

Go ahead, I am not working on it any more.

**Comment by bfabio** [comment]
@maurovitale You can get those files [here](https://github.com/italia/developers.italia.it/issues/692#issuecomment-698877968).
